### PR TITLE
SUMO-251500: Adding monitor's information to OTEL Apps Set2

### PR DESCRIPTION
**Pull request by chetanchoudhary-sumo**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/databases/opentelemetry/couchbase-opentelemetry.md [hunk 1]
```diff
@@ -226,3 +226,20 @@ Use this dashboard to:
...
+
+## Create monitors for Couchbase app
+
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### Couchbase alerts
+
+| Name | Description | Alert Condition | Recover Condition |
```
Comment (amee-sumo) [inline docs/integrations/databases/opentelemetry/couchbase-opentelemetry.md:238]:
```suggestion
| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition |
```

File: docs/integrations/databases/opentelemetry/mariadb-opentelemetry.md [hunk 2]
```diff
@@ -248,3 +248,19 @@ Use this dashboard to:
...
+
+## Create monitors for MariaDB app
+
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### MariaDB alerts
+
+| Name | Description | Alert Condition | Recover Condition |
```
Comment (amee-sumo) [inline docs/integrations/databases/opentelemetry/mariadb-opentelemetry.md:260]:
```suggestion
| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition |
```

File: docs/integrations/web-servers/opentelemetry/squid-proxy-opentelemetry.md [hunk 3]
```diff
@@ -194,3 +194,18 @@ The **The Squid Proxy - HTTP Response Analysis** dashboard provides insights int
...
+
+## Create monitors for SquidProxy app
+
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### SquidProxy alerts
+
+| Name | Description | Alert Condition | Recover Condition |
```
Comment (amee-sumo) [inline docs/integrations/web-servers/opentelemetry/squid-proxy-opentelemetry.md:206]:
```suggestion
| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition |
```

File: docs/integrations/databases/opentelemetry/couchbase-opentelemetry.md (hunk 1 above)
Comment (JV0812) [reply docs/integrations/databases/opentelemetry/couchbase-opentelemetry.md:238]: @amee-sumo We do not want the "Trigger Type (Critical / Warning / MissingData)" column here, as we are only showing the Alert and Recover conditions here.

File: docs/integrations/databases/opentelemetry/mariadb-opentelemetry.md (hunk 2 above)
Comment (JV0812) [reply docs/integrations/databases/opentelemetry/mariadb-opentelemetry.md:260]: @amee-sumo We do not want the "Trigger Type (Critical / Warning / MissingData)" column here, as we are only showing the Alert and Recover conditions here.

File: docs/integrations/web-servers/opentelemetry/varnish-opentelemetry.md [hunk 4]
```diff
@@ -184,3 +184,17 @@ The **Varnish - Visitor Traffic Insight** dashboard provides detailed informatio
...
+
+## Create monitors for Varnish app
+
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### Varnish alerts
+
+| Name | Description | Alert Condition | Recover Condition |
```
Comment (amee-sumo) [inline docs/integrations/web-servers/opentelemetry/varnish-opentelemetry.md:196]:
```suggestion
| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition |
```

File: docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md [hunk 5]
```diff
@@ -184,3 +184,21 @@ Use this dashboard to:
...
+
+## Create monitors for SQL Server Linux app
+
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### SQL Server Linux alerts
+
+| Name | Description | Alert Condition | Recover Condition |
```
[the remark this answers is not in the document]
Comment (JV0812) [reply docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md:196]: @amee-sumo We do not want the "Trigger Type (Critical / Warning / MissingData)" column here, as we are only showing the Alert and Recover conditions here.

File: docs/integrations/databases/opentelemetry/couchbase-opentelemetry.md [hunk 6]
```diff
@@ -226,3 +226,20 @@ Use this dashboard to:
...
+|:--|:--|:--|:--|
+| `Couchbase - Bucket Not Ready` | This alert is triggered when a bucket in the Couchbase cluster is not ready. | Count `>` 0 | Count `<=` 0 |
+| `Couchbase - High Latency HTTP Requests` | This alert is triggered on high average latency for HTTP requests to Couchbase | Count `>` 1000 | Count `<=` 1000 |
```
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/couchbase-opentelemetry.md:241]:
```suggestion
| `Couchbase - High Latency HTTP Requests` | This alert is triggered on high average latency for HTTP requests to the Couchbase. | Count `>` 1000 | Count `<=` 1000 |
```

**Review by amee-sumo — CHANGES_REQUESTED** [review]
Just minor fixes. I've changed the table headings as per our style guide.

File: docs/integrations/databases/opentelemetry/mariadb-opentelemetry.md [hunk 7]
```diff
@@ -248,3 +248,19 @@ Use this dashboard to:
...
+|:--|:--|:--|:--|
+| `MariaDB - Critical Errors` | This alert is triggered when there are critical database errors. | Count `>` 10 | Count `<=` 10 |
+| `MariaDB - Excessive Slow Query Detected` | This alert is triggered when the average time to execute a query is more than 15 seconds for a 5 minute time interval. | Count `>=` 1 | Count `<` 1 |
+| `MariaDB - Failed Login Attempts` | This alert is triggered when there are excessive failed login attempts in a short period. | Count `>=` 1 | Count `<` 1 |
+| `MariaDB - Instance down` | This alert is triggered when we detect that a MariaDB instance is down | Count `>=` 1 | Count `<` 1 |
```
Comment (JV0812) [inline docs/integrations/databases/opentelemetry/mariadb-opentelemetry.md:265]:
```suggestion
| `MariaDB - Instance down` | This alert is triggered when the MariaDB instance is down. | Count `>=` 1 | Count `<` 1 |
```

File: docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md [hunk 8]
```diff
@@ -184,3 +184,21 @@ Use this dashboard to:
...
+|:--|:--|:--|:--|
+| `SQL Server - AppDomain` | This alert is triggered when we detect AppDomain related issues in your SQL Server instance. | Count `>=` 1 | Count `<` 1 |
+| `SQL Server - Backup Fail` | This alert is triggered when we detect that the SQL Server backup failed. | Count `>=` 1 | Count `<` 1 |
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md:199]:
```suggestion
| `SQL Server - Backup Fail` | This alert is triggered when the SQL Server backup is failed. | Count `>=` 1 | Count `<` 1 |
```

File: docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md [hunk 9]
```diff
@@ -184,3 +184,21 @@ Use this dashboard to:
...
+| `SQL Server - Deadlock` | This alert is triggered when we detect deadlocks in a SQL Server instance. | Count `>` 5 | Count `<=` 5 |
+| `SQL Server - Instance Down` | This alert is triggered when we detect that the SQL Server instance is down for 5 minutes. | Count `>` 0 | Count `<=` 0 |
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md:201]:
```suggestion
| `SQL Server - Instance Down` | This alert is triggered when the SQL Server instance is down for 5 minutes. | Count `>` 0 | Count `<=` 0 |
```

File: docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md [hunk 10]
```diff
@@ -184,3 +184,21 @@ Use this dashboard to:
...
+| `SQL Server - Insufficient Space` | This alert is triggered when SQL Server instance could not allocate a new page for database because of insufficient disk space in filegroup. | Count `>` 0 | Count `<=` 0 |
+| `SQL Server - Login Fail` | This alert is triggered when we detect that the user cannot login to SQL Server. | Count `>=` 1 | Count `<` 1 |
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md:203]:
```suggestion
| `SQL Server - Login Fail` | This alert is triggered when the user is unable to login to the SQL Server. | Count `>=` 1 | Count `<` 1 |
```

File: docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md [hunk 11]
```diff
@@ -184,3 +184,21 @@ Use this dashboard to:
...
+| `SQL Server - Mirroring Error` | This alert is triggered when we detect that the SQL Server mirroring has error. | Count `>=` 1 | Count `<` 1 |
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md:204]:
```suggestion
| `SQL Server - Mirroring Error` | This alert is triggered when the SQL Server mirroring has error. | Count `>=` 1 | Count `<` 1 |
```

File: docs/integrations/web-servers/opentelemetry/squid-proxy-opentelemetry.md [hunk 12]
```diff
@@ -194,3 +194,18 @@ The **The Squid Proxy - HTTP Response Analysis** dashboard provides insights int
...
+|:--|:--|:--|:--|
+| `Squid Proxy - High Client (HTTP 4xx) Error Rate` | This alert is triggered when there are too many HTTP requests (>5%) with a response status of 4xx. | Count `>` 0 | Count `<=` 0 |
+| `Squid Proxy - High Denied Request` | This alert is triggered when there are too many HTTP denied requests (>5%) | Count `>` 0 | Count `<=` 0 |
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/squid-proxy-opentelemetry.md:209]:
```suggestion
| `Squid Proxy - High Denied Request` | This alert is triggered when there are too many HTTP denied requests (>5%). | Count `>` 0 | Count `<=` 0 |
```

File: docs/integrations/databases/opentelemetry/oracle-opentelemetry.md [hunk 13]
```diff
@@ -559,3 +559,26 @@ See information derived from the syslog audit trail, including successful and fa
...
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### Oracle alerts
+
+| Name | Description | Alert Condition | Recover Condition |
+|:--|:--|:--|:--|
+| `Oracle - Admin Restricted Command Execution` | This alert is triggered when the Listener could not resolve a command. | Count `>` 0 | Count `<=` 0 |
+| `Oracle - Archival Log Creation` | This alert is triggered when there is an archive log creation error. | Count `>` 0 | Count `<=` 0 |
```
Comment (amee-sumo) [inline docs/integrations/databases/opentelemetry/oracle-opentelemetry.md:574]:
```suggestion
| `Oracle - Archival Log Creation` | This alert is triggered when an archive log creation error occurs. | Count `>` 0 | Count `<=` 0 |
```

File: docs/integrations/databases/opentelemetry/oracle-opentelemetry.md [hunk 14]
```diff
@@ -559,3 +559,26 @@ See information derived from the syslog audit trail, including successful and fa
...
+
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### Oracle alerts
+
+| Name | Description | Alert Condition | Recover Condition |
+|:--|:--|:--|:--|
+| `Oracle - Admin Restricted Command Execution` | This alert is triggered when the Listener could not resolve a command. | Count `>` 0 | Count `<=` 0 |
```
Comment (amee-sumo) [inline docs/integrations/databases/opentelemetry/oracle-opentelemetry.md:573]:
```suggestion
| `Oracle - Admin Restricted Command Execution` | This alert is triggered when the Listener cannot resolve a command. | Count `>` 0 | Count `<=` 0 |
```

File: docs/integrations/databases/opentelemetry/oracle-opentelemetry.md [hunk 15]
```diff
@@ -559,3 +559,26 @@ See information derived from the syslog audit trail, including successful and fa
...
+| `Oracle - Archival Log Creation` | This alert is triggered when there is an archive log creation error. | Count `>` 0 | Count `<=` 0 |
+| `Oracle - Block Corruption` | This alert is triggered when we detect corrupted data blocks. | Count `>` 0 | Count `<=` 0 |
```
Comment (amee-sumo) [inline docs/integrations/databases/opentelemetry/oracle-opentelemetry.md:575]:
```suggestion
| `Oracle - Block Corruption` | This alert is triggered when corrupt data blocks are detected. | Count `>` 0 | Count `<=` 0 |
```

File: docs/integrations/databases/opentelemetry/oracle-opentelemetry.md [hunk 16]
```diff
@@ -559,3 +559,26 @@ See information derived from the syslog audit trail, including successful and fa
...
+| `Oracle - Database Crash` | This alert is triggered when the database crashes. | Count `>` 0 | Count `<=` 0 |
+| `Oracle - Deadlock` | This alert is triggered when deadlocks are detected. | Count `>` 5 | Count `<=` 5 |
+| `Oracle - Fatal NI Connect Error` | This alert is triggered when we detect a "Fatal NI connect error". | Count `>` 0 | Count `<=` 0 |
```
Comment (amee-sumo) [inline docs/integrations/databases/opentelemetry/oracle-opentelemetry.md:578]:
```suggestion
| `Oracle - Fatal NI Connect Error` | This alert is triggered when a "Fatal NI connect error" is detected. | Count `>` 0 | Count `<=` 0 |
```

File: docs/integrations/databases/opentelemetry/oracle-opentelemetry.md [hunk 17]
```diff
@@ -559,3 +559,26 @@ See information derived from the syslog audit trail, including successful and fa
...
+| `Oracle - Internal Errors` | This alert is triggered when internal errors are detected. | Count `>` 0 | Count `<=` 0 |
+| `Oracle - Login Fail` | This alert is triggered when we detect that a user cannot login. | Count `>` 0 | Count `<=` 0 |
```
Comment (amee-sumo) [inline docs/integrations/databases/opentelemetry/oracle-opentelemetry.md:580]:
```suggestion
| `Oracle - Login Fail` | This alert is triggered when a user login failure is detected. | Count `>` 0 | Count `<=` 0 |
```

File: docs/integrations/databases/opentelemetry/oracle-opentelemetry.md [hunk 18]
```diff
@@ -559,3 +559,26 @@ See information derived from the syslog audit trail, including successful and fa
...
+| `Oracle - Possible Inappropriate Activity` | This alert is triggered when we detect possible inappropriate activity. | Count `>` 0 | Count `<=` 0 |
```
Comment (amee-sumo) [inline docs/integrations/databases/opentelemetry/oracle-opentelemetry.md:581]:
```suggestion
| `Oracle - Possible Inappropriate Activity` | This alert is triggered when possible inappropriate activity is detected. | Count `>` 0 | Count `<=` 0 |
```

File: docs/integrations/databases/opentelemetry/oracle-opentelemetry.md [hunk 19]
```diff
@@ -559,3 +559,26 @@ See information derived from the syslog audit trail, including successful and fa
...
+| `Oracle - TNS Error` | This alert is triggered when we detect TNS operations errors. | Count `>` 0 | Count `<=` 0 |
+| `Oracle - Unable To Extend Tablespace` | This alert is triggered when we detect that we are unable to extend tablespaces. | Count `>` 0 | Count `<=` 0 |
```
Comment (amee-sumo) [inline docs/integrations/databases/opentelemetry/oracle-opentelemetry.md:583]:
```suggestion
| `Oracle - Unable To Extend Tablespace` | This alert is triggered when tablespace extension failures are detected. | Count `>` 0 | Count `<=` 0 |
```

File: docs/integrations/databases/opentelemetry/oracle-opentelemetry.md [hunk 20]
```diff
@@ -559,3 +559,26 @@ See information derived from the syslog audit trail, including successful and fa
...
+| `Oracle - Unauthorized Command Execution` | This alert is triggered when we detect that a user is not authorized to execute a requested listener command in a Oracle instance. | Count `>` 0 | Count `<=` 0 |
```
Comment (amee-sumo) [inline docs/integrations/databases/opentelemetry/oracle-opentelemetry.md:584]:
```suggestion
| `Oracle - Unauthorized Command Execution` | This alert is triggered when a user is not authorized to execute a requested listener command in an Oracle instance. | Count `>` 0 | Count `<=` 0 |
```

File: docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md [hunk 21]
```diff
@@ -184,3 +184,21 @@ Use this dashboard to:
...
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### SQL Server Linux alerts
+
+| Name | Description | Alert Condition | Recover Condition |
+|:--|:--|:--|:--|
+| `SQL Server - AppDomain` | This alert is triggered when we detect AppDomain related issues in your SQL Server instance. | Count `>=` 1 | Count `<` 1 |
+| `SQL Server - Backup Fail` | This alert is triggered when the SQL Server backup is failed. | Count `>=` 1 | Count `<` 1 |
```
Comment (amee-sumo) [inline docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md:199]:
```suggestion
| `SQL Server - Backup Fail` | This alert is triggered when the SQL Server backup fails. | Count `>=` 1 | Count `<` 1 |
```

**Review by amee-sumo — CHANGES_REQUESTED** [review]
Have followed a similar grammatical structure (parallelism) across all descriptions in the Oracle alerts table. Please maintain the same for the rest of the sections.

**Review by amee-sumo — APPROVED** [review]
Have maintained parallelism across all descriptions in all the tables.
Rest LGTM

**Pull request merged**